templates/page.hbs

```diff
@@ -13,8 +13,16 @@
   </head>
   <body>
     {{!-- Header goes here --}}
-
+    
     <main>
+      {{#each header}}
+        {{{module this}}}
+      {{/each}}
+
+       {{#each video}}
+        {{{module this}}}
+      {{/each}}
```

Comment:
remove this block (from line 22 to 24) !!
The video component is already inside of the` {{#each components}} `loop

Reply:
Done.